### PR TITLE
Add MVSNeRF and GNeRF

### PR DESCRIPTION
**Pull request by apchenstu**
# Pull requests instruction

1. Put the bibtex in `awesome-NeRF/citations/<YOUR-PROJECT>.txt`.
2. Modify the README and follow the format TITLE, AUTHOR, CONFERENCE YEAR | OPTIONAL LINK | BIBTEX. For example:
- [NeRF: Representing Scenes as Neural Radiance Fields for View Synthesis](https://www.matthewtancik.com/nerf), Mildenhall et al., ECCV 2020 | [github](https://github.com/bmild/nerf)  | [bibtex](./citations/nerf.txt)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,17 +53,23 @@ A curated list of awesome neural radiance fields papers, inspired by [awesome-co
 - [Learned Initializations for Optimizing Coordinate-Based Neural Representations](https://arxiv.org/abs/2012.02189), Tancik et al., CVPR 2021 | [github](https://github.com/tancik/learnit) | [bibtex](./NeRF-and-Beyond.bib#L268-L274) <!---Tancik20arxiv_meta-->
 - [pi-GAN: Periodic Implicit Generative Adversarial Networks for 3D-Aware Image Synthesis](https://marcoamonteiro.github.io/pi-GAN-website/), Chan et al., CVPR 2021 | [bibtex](./NeRF-and-Beyond.bib#L24-L30) <!---Chan20arxiv_piGAN-->
 - [Portrait Neural Radiance Fields from a Single Image](https://portrait-nerf.github.io/), Gao et al., Arxiv 2020 | [bibtex](./NeRF-and-Beyond.bib#L95-L101) <!---Gao20arxiv_pNeRF-->
+- [ShaRF: Shape-conditioned Radiance Fields from a Single View](https://arxiv.org/pdf/2102.08860.pdf), Rematas et al., Arxiv 2021 | [bibtex](./citations/sharf.txt)
```
Comment (yenchenlin) [inline README.md:56]: Can you remove the change for this line?

File: README.md [hunk 2]
```diff
@@ -53,17 +53,23 @@ A curated list of awesome neural radiance fields papers, inspired by [awesome-co
...
+- [PVA: Pixel-aligned Volumetric Avatars](https://volumetric-avatars.github.io/), Raj et al., Arxiv 2021 | [bibtex](./citations/pva.txt)
```
Comment (yenchenlin) [inline README.md:57]: Also remove this change.

File: README.md [hunk 3]
```diff
@@ -53,17 +53,23 @@ A curated list of awesome neural radiance fields papers, inspired by [awesome-co
...
+- [IBRNet: Learning Multi-View Image-Based Rendering](https://ibrnet.github.io/static/paper.pdf), Wang et al., Arxiv 2021 | [github](https://github.com/googleinterns/IBRNet) | [bibtex](./citations/ibr.txt)
```
Comment (yenchenlin) [inline README.md:58]: And also remove this change

**Review by yenchenlin — COMMENTED** [review]
@apchenstu Thanks a lot, these are nice papers. Can you change your PR slightly according to my comments? Then I will merge it!

**Comment by apchenstu** [comment]
Great thanks for your recognition, I have created a new pull application so I close this one, thanks~

**Pull request closed without merging**